src/app/pages/task/task.page.scss

```diff
@@ -1,24 +1,59 @@
 .task-item {
-    padding-bottom: 0;
+  padding-bottom: 0;
 }
 
 .item .sc-ion-label-md-h {
-    overflow: visible;
-    white-space: pre-wrap;
+  overflow: visible;
+  white-space: pre-wrap;
 }
 
 ion-note {
-    padding: 10px;
-    line-height: 21px;
-    white-space: pre-wrap;
+  padding: 10px;
+  line-height: 21px;
+  white-space: pre-wrap;
 }
 
 ion-badge {
-    vertical-align: middle;
-    white-space: pre-line;
-    text-align: left;
+  vertical-align: middle;
+  white-space: pre-line;
+  text-align: left;
 }
 
 ion-footer div {
-    margin: 10px;
-}
\ No newline at end of file
+  margin: 10px;
+}
+
+.create-button {
+  margin-left:15px;
+}
+
+.trash-button {
+  margin-left:5px;
+}
+
+.network-badge {
+  float: right;
+  height: calc(2.1em + 4px);
+  font-size: 13px;
+  text-transform: uppercase;
+  vertical-align: middle;
+  line-height: calc(2.1em + 4px);
+  --padding-top: 0;
```

Comment:
Not sure if they valid saas properties. 
Github is flagging that so let's check if that is making any difference in layout.

Reply:
My bad! ignore this.